lib/modules/storage/storageProcessesLauncher.js

```diff
@@ -109,13 +109,21 @@ class StorageProcessesLauncher {
         silent: self.logger.logLevel !== 'trace',
         exitCallback: self.processExited.bind(this, storageName)
       });
-      self.processes[storageName].send({
-        action: constants.blockchain.init, options: {
-          storageConfig: self.storageConfig,
-          blockchainConfig: self.blockchainConfig,
-          cors: self.buildCors()
-        }
+      this.events.once("web3Ready", () => {
+        this.events.request("blockchain:object", (blockchain) => {
+          blockchain.determineDefaultAccount((err, defaultAccount) => {
```

Comment:
```suggestion
          blockchain.determineDefaultAccount((err, defaultAccount) => {
            if (err) {
              return callback(err);
            }
```
Just need to double check that callback can't be called twice

Reply:
Fixed up.